code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2173,3 +2173,34 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		M.adjust_fire_stacks(1)
 		M.IgniteMob()
 	..()
+
+/datum/reagent/consumable/ethanol/customizable
+	name = "Ale"
+	description = "An alcoholic beverage brewed by hand."
+	color = "#664300" // rgb: 102, 67, 0
+	nutriment_factor = 1 * REAGENTS_METABOLISM
+	boozepwr = 0 // filled out by brewing
+	taste_description = "ale"
+	glass_name = "glass of ale"
+	glass_desc = "A glass of ale. You feel a fey mood coming on just looking at it."
+	var/datum/reagents/contained_reagents
+
+/datum/reagent/consumable/ethanol/customizable/New()
+	..()
+	contained_reagents = new/datum/reagents(10000)
```

Comment:
I will say, I am questioning whether or not the contained reagents system is a good idea to keep moving forward with. I'll leave it to actual maintainers to decide, but keep in mind that this work may be scrapped. I do like the rest of the custom alcohol system, though.

Reply:
This will not be scrapped, as it is the entire point of the gimmick.